lib/generators/rolemodel/linters/rubocop/templates/app/cops/form_error_response.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+# There are three levels of matching here:
+# Methods (def on_if)
+# This scans code and only stops on the method specified.
+# E.G. `on_if` will only move into the below checks when it encounters an if statement.
+
+# Node matcher (def_node_matcher :form_error)
+# This scans the block of code (in this case, the else branch of the if block)
+# using AST (Abstract Tree Syntax) to match certain patterns
+
+# Regex
+# This checks the specific matching line to ensure it meets our expectations
+
+# Custom Cop to Ensure Form Error Response Statuses
+class FormErrorResponse < RuboCop::Cop::Cop
```

Comment:
https://docs.rubocop.org/rubocop/v1_upgrade_notes.html#base-class

```suggestion
class FormErrorResponse < RuboCop::Cop::Base
```
RuboCop::Cop::Base has been deprecated:
Use RuboCop::Cop::Base instead.